src/trufont/objects/factories.py

```diff
@@ -97,6 +100,36 @@ def pathGraphicsPathFactory(path):
             if point.type == "curve":
                 graphicsPath.AddCurveToPoint(*stack)
                 stack = []
+            elif point.type == "qcurve" or (not open_ and point is points[-1]):
```

Comment:
what's (not open_ and point is points[-1]) ? what should the last point be assumed to be qcurve?

Reply:
It's part of the all-off-curve handling. Remove completely?